src/lib/types/experimental.ts

```diff
@@ -208,6 +209,19 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_EXECUTIVE_DASHBOARD,
         false,
     ),
+    feedbackComments: {
+        name: 'feedbackComments',
+        enabled: parseEnvVarBoolean(
+            process.env.UNLEASH_EXPERIMENTAL_FEEDBACK_COMMENTS,
+            false,
+        ),
+        payload: {
```

Comment:
nice. is it a first time we do sth like this in our code?

Reply:
No, we also do this for the message banner. I took the pattern from there.